### ISSUE TITLE
@3x (for iPhone 6 Plus)

### ISSUE THREAD

**Issue by nikbelikov** [imulus/retinajs#205]
And what about iPhone 6 Plus which has 401 ppi?

**Comment by amberweinberg** [comment]
I'd like to know too

**Comment by ChizHuang** [comment]
Same here. Does it support 3x?

**Comment by llahnoraa** [comment]
@amberweinberg you are correct. iPhone 6 Plus is the only one that does support 3x. [From StackOverFlow: Does OS X support 3x scaling yet?](http://stackoverflow.com/questions/26696949/does-os-x-support-3x-scaling-yet)
